src/app/footer/footer.component.scss

```diff
@@ -17,6 +17,10 @@ footer {
 	}
 
 	@media(max-width: 400px) {
-		justify-content: space-around;
+		justify-content: flex-start;
+
+		div a {
```

Comment:
Isn't there a better way to do this? The problem with this is the fact it'll affect every `a`-s that are inside `div`-s. I think you could just create a class or specify it's only for a particular class. Does this also affect other links?